spec/features/idv/doc_auth/socure_document_capture_spec.rb

```diff
@@ -91,6 +91,99 @@
         end
       end
 
+      context 'reuses valid capture app urls when appropriate', allow_browser_log: true do
+        before do
+          allow(IdentityConfig.store).to receive(:doc_auth_max_attempts).and_return(max_attempts)
+          (max_attempts - 1).times do
+            socure_docv_upload_documents(docv_transaction_token: @docv_transaction_token)
+          end
+        end
+
+        context 'successfully erases capture app url when flow is complete' do
+          before do
+            DocAuth::Mock::DocAuthMockClient.reset!
+          end
+
+          it 'proceeds to the next page with valid info' do
+            document_capture_session = DocumentCaptureSession.find_by(user_id: @user.id)
+            expect(page).to have_current_path(fake_socure_document_capture_app_url)
+            visit idv_socure_document_capture_path
+            expect(page).to have_current_path(idv_socure_document_capture_path)
+            document_capture_session.reload
+            expect(document_capture_session.socure_docv_capture_app_url).
+              to eq(fake_socure_document_capture_app_url)
+            socure_docv_upload_documents(
+              docv_transaction_token: @docv_transaction_token,
+            )
+            document_capture_session.reload
+            expect(document_capture_session.socure_docv_capture_app_url).to be_nil
+          end
+
+          it 'submits front ID, exits app, reuse capture app url' do
+            document_capture_session = DocumentCaptureSession.find_by(user_id: @user.id)
+            expect(page).to have_current_path(fake_socure_document_capture_app_url)
+            visit idv_socure_document_capture_path
+            expect(page).to have_current_path(idv_socure_document_capture_path)
+            document_capture_session.reload
+            expect(document_capture_session.socure_docv_capture_app_url).
+              to eq(fake_socure_document_capture_app_url)
+            socure_docv_send_webhook(
+              docv_transaction_token: @docv_transaction_token,
+              event_type: 'DOCUMENT_FRONT_UPLOADED',
+            )
+            document_capture_session.reload
+            expect(document_capture_session.socure_docv_capture_app_url).
+              to eq(fake_socure_document_capture_app_url)
+            visit idv_socure_document_capture_path
+            expect(page).to have_current_path(idv_socure_document_capture_path)
+            document_capture_session.reload
+            expect(document_capture_session.socure_docv_capture_app_url).
+              to eq(fake_socure_document_capture_app_url)
+          end
```

Comment:
seems like these specs can be merged into 1 to help our spec runtime. 

Also, as is the stub just has 1 capture app url so we do not have another url to compare it to to see if it changes from one url to another? 🤔